.gitignore

```diff
@@ -0,0 +1,17 @@
+
+# JetBrains IDEs
+/.idea/
+
+# VS Code
+.vscode/*
+!.vscode/settings.json
+!.vscode/tasks.json
+!.vscode/launch.json
+!.vscode/extensions.json
+!.vscode/*.code-snippets
```

Comment:
@xmfcx Do you want to add these json and snippets files?
If not, I think we can simplify this.

Also, can I make another PR for `.gitignore` to make the diff easy to understand?

Reply:
https://github.com/autowarefoundation/autoware-documentation/pull/30